include/klee/ExecutionState.h

```diff
@@ -71,7 +79,7 @@ class ExecutionState {
   // unsupported, use copy constructor
   ExecutionState &operator=(const ExecutionState &);
 
-  std::map<std::string, std::string> fnAliases;
+  std::vector<FunctionAlias> fnAliases;
```

Comment:
Have a vector of references to `FunctionAlias` here,
e.g. `std::vector<ref<FunctionAlias> >`.
This it's shared between states and we don't make a deep copy of the strings all the time.